c7n/filters/core.py

```diff
@@ -335,7 +335,12 @@ def process_value_type(self, sentinel, value):
                 sentinel = datetime.now(tz=tzutc()) - timedelta(sentinel)
 
             if not isinstance(value, datetime):
-                value = parse(value)
+                # EMR bug when testing ages in EMR. This is due to
+                # EMR not having more functionality.
+                try:
+                    value = parse(value)
```

Comment:
i'm a bit nervous about this one, i think we need some default handling here that does something better by making it configurable, ie. does a given policy want to skip missing values, or use a default value. the blanket value = 0 as can lead to some unintended consequences when reasoning about policies.